### ISSUE TITLE
Carousel buttons must have discernible text

### ISSUE THREAD

**Issue by Bryelmo** [primefaces/primeng#12924]
### Describe the feature you would like to see added

It would be better for carousel component pass the accessibility WCAG 2.1 AA standard test. I'm using it in a project that requires this standard.

### Is your feature request related to a problem?

Carousel component fails a WCAG 2.1 AA standard test with Axe Devtools and Accessibility insights for web extensions on Google Chrome browser.

`<button type="button" pripple="" class="p-ripple p-element p-carousel-prev p-link p-disabled ng-star-inserted" ng-reflect-ng-class="[object Object]" disabled=""><span ng-reflect-ng-class="[object Object]" class="p-carousel-prev-icon pi pi-chevron-left"></span><span class="p-ink"></span></button>`

### Describe the solution you'd like

Axe Devtools says how to solve this problem, you need to fix at least (1) of the following:

- Element does not have inner text that is visible to screen readers
- aria-label attribute does not exist or is empty
- aria-labelledby attribute does not exist, references elements that do not exist or references elements that are empty
- Element has no title attribute
- Element's default semantics were not overridden with role="none" or role="presentation"

**Comment by mertsincan** [comment]
Hi,

So sorry for the delayed response! Improvements have been made to many components recently, both in terms of performance and enhancement. Therefore, this improvement may have been developed in another issue ticket without realizing it. You can check this in the documentation. If there is no improvement on this, can you open a new issue so we can include it in our roadmap?

Thanks a lot for your understanding!
Best Regards,
